### ISSUE TITLE
Cannot read property 'isTransactionActive' of undefined

### ISSUE THREAD

**Issue by goldcaddy77** [vesper-framework/vesper#19]
Hi, I'm trying to get a new project up and running by doing the following (on version `0.1.7`):

```
npm i vesper -g
vesper init --name my-project --typescript
cd my-project
yarn && yarn start
open http://localhost:3000/playground
```

I try the following query:

```
query {
  users {
    id
    firstName
  }
}
```

...and I get:

```
TypeError: Cannot read property 'isTransactionActive' of undefined
    at /Users/dancaddigan/Code/tmp/my-project/src/index.ts:140:53
```

Any thoughts?  I get the same thing if I change the DB engine to Postgres and create a new pg DB.

**Comment by pleerock** [comment]
I just fixed this issue, please check it in `0.1.8`

**Comment by goldcaddy77** [comment]
Cool, thanks 👍
